jina/peapods/runtimes/asyncio/grpc/__init__.py

```diff
@@ -28,3 +30,4 @@ async def async_cancel(self):
 
     async def async_run_forever(self):
         await self.server.wait_for_termination()
```

Comment:
should there be a `try catch`?

Reply:
no 
![image](https://user-images.githubusercontent.com/2041322/106897342-16c49480-66f3-11eb-8a5e-a15888d30256.png)
https://docs.jina.ai/api/jina.peapods.runtimes.base.html?highlight=baseruntime#jina.peapods.runtimes.base.BaseRuntime